docs/sources/administration/provisioning/index.md

```diff
@@ -81,7 +81,7 @@ The configuration file can also list data sources to automatically delete, calle
 Grafana deletes the data sources listed in `deleteDatasources` _before_ adding or updating those in the `datasources` list.
 
 You can configure Grafana to automatically delete provisioned data sources when they're removed from the provisioning file.
-To do so, add `prune: true` to the root of your provisioning file.
+To do so, add `prune: true` to your datasource provisioning file.
```

Comment:
Thanks for trying to make this more clear.

I'm thinking that really it's the previous sentence and this paragraph as a whole that explains the process. I'm not sure this specific sentence has much of a different meaning than the previous one.

Is the belief that "root of your provisioning file" is being confused for "a file in the root of your provisioning directory"?

```suggestion
To do so, add `prune: true` to the root of your data source provisioning file.
```

Reply:
> Is the belief that "root of your provisioning file" is being confused for "a file in the root of your provisioning directory"?

yeah, the "root" word is the one that could be confused, but we say `root of .. file` so it should be clear enough 🤷